ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml

```diff
@@ -126,9 +132,24 @@ objects:
 
         # test specific code
         # don't let clients impact the global kubeconfig
-        mkdir -p /go/src/github.com/openshift/
-        cp -r /tmp/shared/openshift-azure /go/src/github.com/openshift/
-        cd /go/src/github.com/openshift/openshift-azure/
+        if [[ "${TEST_IMAGE}" == *"/bushslicer"* ]]; then
+          TEST_NODE_OPTS="etcd:master:node"
+          TEST_CLUSTER="${RESOURCEGROUP}.eastus.cloudapp.azure.com"
+          export TESTENV="BUSHSLICER_V3"
+          export BUSHSLICER_DEFAULT_ENVIRONMENT=online_starter
+          export BUSHSLICER_DEBUG_AFTER_FAIL=1
```

Comment:
Since we're not doing active development, and this is in ci/cd, we can remove the DEBUG_AFTER_FAIL line here, or set it to 0.